frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -558,14 +558,11 @@ function ReaderHighlight:addToMainMenu(menu_items)
                 icon = "texture-box",
                 ok_callback = function()
                     local count = 0
-                    for _, items in pairs(self.view.highlight.saved) do
-                        if items then
-                            count = count + #items
-                            for i = 1, #items do
-                                local item = items[i]
-                                item.drawer = self.view.highlight.saved_drawer
-                                item.color = self.view.highlight.saved_color
-                            end
+                    for _, item in ipairs(self.ui.annotation.annotations) do
+                        if item.drawer then
```

Comment:
Does it need to skip for "inverted"?
Depends on what color do we want after changing "inverted" to smth else.

Reply:
The intent is to apply *both* style & colors, so it doesn't make sense to special-cases anything right now.

(i.e., both drawer & color are updated).

Reply:
Got it.
If we apply "inverted". the `color` field is written anyway?

Reply:
That's a very good question ;p.

Probably couldn't hurt to avoid doing that ;).

Reply:
Yeah, we always populate the color field, it's just ignored by ReaderView for `invert` ;).